### ISSUE TITLE
0 Hashrate no error

### ISSUE THREAD

**Issue by makipsee** [Lolliedieb/lolMiner-releases#666]
LOLMiner v1.28
Windows 10 Pro
GPU's: XFX 5700 XT,  EVGA 1070 TI and ASUS 1070 TI

I have switched from PhoenixMiner to LOLMiner for 3 of my cards because of the dag repair feature. with that being said i am having a random random issue where one card will drop to 0 hashrate while mining at random times the EVGA 1070 will just stop mining and hashrate will drop to 0 but will not trigger a problem and will keep mining without the one card. i have tried adding the arg "--watchdog exit" but it doesnt cause a restart. 

 i also have 2 2060 KO's on this system but LOLminer will not work with them. i get the error 
 "error when allocating memory or building DAG. Disabling card."

**Comment by jgonzis** [comment]
The error happen because you need to increase the virtual  disk to 60000 - 60000

**Comment by makipsee** [comment]
Sorry I forgot to mention that. When I first got the error my virtual memory was set to 40000. After searching I came across something on here on someone else's post about changing it to 60000 and I have since changed it to 60000 but same error. And while the miners are running the computer is only using about 36000 of the virtual memory

**Comment by jgonzis** [comment]
> Sorry I forgot to mention that. When I first got the error my virtual memory was set to 40000. After searching I came across something on here on someone else's post about changing it to 60000 and I have since changed it to 60000 but same error. And while the miners are running the computer is only using about 36000 of the virtual memory

Can you try to put some photo of the start detection to know and have more information... try more than 60000 put 80000 or 100000... or try free the Linux OS like MMPOS is free and special for mining...

**Comment by makipsee** [comment]
I'll try and get some pictures. I can't only have a 120GB drive on it and 60000 is the max I can make it.

**Comment by jgonzis** [comment]
> I'll try and get some pictures. I can't only have a 120GB drive on it and 60000 is the max I can make it.

Try in a USB MMPOS.EU or HiveOS... that is a fast way to have Linux that is the best OS for mining

**Comment by makipsee** [comment]
I tried Linux but they don't have away of controlling voltage on Nvidia cards like Afterburner

**Comment by jgonzis** [comment]
> I tried Linux but they don't have away of controlling voltage on Nvidia cards like Afterburner

That's not true... only Ampere and Turing you can do it, both Linux OS have it....

**Comment by makipsee** [comment]
everywhere i looked it said it couldnt be done...

**Comment by makipsee** [comment]
ok so for this i only selected the 2 2060 cards 
![Clipboard02](https://user-images.githubusercontent.com/87274418/125213028-ea530c80-e27e-11eb-8d0d-375a372e4033.jpg)
![Clipboard03](https://user-images.githubusercontent.com/87274418/125213031-ede69380-e27e-11eb-85b2-7d2957294ceb.jpg)

**Comment by jgonzis** [comment]
That is Virtual DISK Problem... if you do it with only 1 GPU with your actual Virtual Disk 60000 it should work. try a Linux OS USB. Is free to try it ...

**Comment by makipsee** [comment]
You don't think it should have worked at that time with 2 GPU's being set to 60000?
